### DISCUSSION TITLE
Is there a way to host different book versions (tags) on the same domain/repo?

### DISCUSSION THREAD

**Discussion by fmaussion** [executablebooks/meta#331] [Q&A]
I am currently hosting a book via gh pages (manual builds and pushes with `ghp-import` because the build is quite expensive) at https://oggm.org/tutorials. 

I was wondering if there is a way to host different versions of a book (e.g. one per git tag), like sphinx + readthedocs is doing? The only way I can think of (with manual builds) is with different repos, but is there maybe another way using CI?

**Comment by choldgraf** [comment]
I think the books are pretty much self contained, so one approach I've seen others follow is to put different versions of the same docs into different sub folders

**Reply by fmaussion** [reply]
Thanks @choldgraf ! 

For this use case (tutorials, closer to a documentation, i.e. tied to a library version number), I'm starting to wonder if I shouldn't have used the sphinx-book-theme + readthedocs instead (although it would come with other challenges in terms of resources since our builds take a looooong time...)

**Reply by choldgraf** [reply]
It should be very similar! There is very little that jupyter book does that sphinx cannot do alone!
